bundles/aero.minova.rcp.form.menu/pom.xml

```diff
@@ -48,7 +48,7 @@
 				<artifactId>maven-surefire-plugin</artifactId>
 				<version>2.22.0</version>
 				<configuration>
-					<argLine>--illegal-access=permit</argLine>
+					<argLine>@{argLine} --illegal-access=permit</argLine>
 				</configuration>
 			</plugin>
 			<plugin>
```

Comment:
In der Implementierung (Testsuite ist in einem anderen Modul) das `maven-surefire-plugin` zu konfigurieren, ist eigentlich Unsinn.